datasets/flwr_datasets/utils.py

```diff
@@ -239,3 +239,68 @@ def _check_division_config_correctness(
 ) -> None:
     _check_division_config_types_correctness(division)
     _check_division_config_values_correctness(division)
+
+
+def concatenate_divisions(
+    partitioner: Partitioner,
+    partition_division: Union[List[float], Tuple[float, ...], Dict[str, float]],
+    division_id: Union[int, str],
+) -> Dataset:
+    """Create a dataset by concatenation the divisions of the partitions.
+
+    The divisions are created based on the `partition_division` and accessed based
+    on the `division_id`. It can be used to create e.g. centralized dataset from
+    federated on-edge test sets.
+    sets
```

Comment:
```suggestion
    """Create a dataset by concatenation all partitions in the same division.

    The divisions are created based on the `partition_division` and accessed based
    on the `division_id`. It can be used to create e.g. centralized dataset from
    federated on-edge test sets.
```

Reply:
Let me know if the first sentence still makes sense to you.